README.md

```diff
@@ -193,7 +193,6 @@ This project uses the [Apache License 2.0](LICENSE.txt).
 
 However there is some code that is included under other licensing as follows:
 
-* The [Gaussian 16 driver](qiskit/chemistry/drivers/gaussiand) in `qiskit.chemistry`
-  contains [work](qiskit/chemistry/drivers/gaussiand/gauopen) licensed under the
-  [Gaussian Open-Source Public License](qiskit/chemistry/drivers/gaussiand/gauopen/LICENSE.txt).
-
+* The [Gaussian 16 driver](qiskit_nature/drivers/gaussiand) in `qiskit.chemistry`
```

Comment:
```suggestion
* The [Gaussian 16 driver](qiskit_nature/drivers/gaussiand) in `Qiskit Nature`
```
Since there is no longer a qiskit.chemistry I think it should perhaps be more like the above instead

Reply:
Included